projects.yml

```diff
@@ -1148,8 +1133,6 @@ staging-xpi-manifest:
   repo: https://github.com/mozilla-releng/staging-xpi-manifest
   repo_type: git
   branches:
-    - name: "*"
-      level: 1
```

Comment:
For the staging / level 1 repos, we may actually want to remove the `main`/`master` branch and keep just `*`?
AFAIK using the default branch is only necessary when we want to change actions, but for everything else using other branches is probably better (less chance of collision)?

Reply:
Hm, I think that's probably reasonable. If we're doing that, we should probably keep a level 1 `*` branch in level 3 repos as well (I keep going back and forth on whether or not that's a good idea...but I don't think it does any harm).

>  AFAIK using the default branch is only necessary when we want to change actions, but for everything else using other branches is probably better (less chance of collision)?

Can you clarify what you mean by "less chance of collision"? I'm not sure I see how it connects to the rest of the comment.

Reply:
I don't think this applies to the same extent to other repos, where there's only a couple of branches we care about and the rest are pretty much only used for PRs, which get their own graphs.

For the staging repos though the entire point is to make arbitrary changes without going through PRs.  I think of those a bit like try, where each push is basically its own mercurial head, and branch names are irrelevant.

Reply:
If you're exclusively talking about staging repos, yeah, I agree those are different. If we're talking about non-staging level 1 repos as well (of which there are many that aren't staging), I'm not sure I see those as any different from level 3 repos.

Reply:
I was thinking about the staging repos specifically, yes.  Sorry for the confusion.

Reply:
Oh sure - yes. Let's do that.